src/fn/move-contacts.js

```diff
@@ -49,19 +73,18 @@ const updateLineagesAndStage = async (options, db) => {
     trace(`Considering primary contact updates to ${ancestors.length} ancestor(s) of contact ${prettyPrintDocument(contactDoc)}.`);
     const updatedAncestors = replaceLineageInAncestors(descendantsAndSelf, ancestors);
 
-    const reportsCreatedByDescendants = await fetch.reportsCreatedBy(db, descendantsAndSelf.map(descendant => descendant._id));
-    trace(`${reportsCreatedByDescendants.length} report(s) created by these affected contact(s) will update`);
-    const updatedReports = replaceLineageInReports(reportsCreatedByDescendants, replacementLineage, contactId);
+    const movedReports = await moveReportsBatch(db, descendantsAndSelf.map(descendant => descendant._id), 0, replacementLineage, contactId, options);
```

Comment:
Please pass a named param instead of one that is calculated inline `descendantsAndSelf.map(descendant => descendant._id)`.